db/schema.rb

```diff
@@ -0,0 +1,91 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2022_09_22_131731) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "add_post_ref_to_comments", force: :cascade do |t|
+    t.bigint "post_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["post_id"], name: "index_add_post_ref_to_comments_on_post_id"
+  end
+
+  create_table "add_post_ref_to_likes", force: :cascade do |t|
+    t.bigint "post_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["post_id"], name: "index_add_post_ref_to_likes_on_post_id"
+  end
+
+  create_table "add_user_ref_to_comments", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_add_user_ref_to_comments_on_user_id"
+  end
+
+  create_table "add_user_ref_to_likes", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_add_user_ref_to_likes_on_user_id"
+  end
+
+  create_table "add_user_ref_to_posts", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_add_user_ref_to_posts_on_user_id"
+  end
+
+  create_table "comments", force: :cascade do |t|
+    t.integer "authorId"
+    t.integer "postId"
+    t.text "text"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "likes", force: :cascade do |t|
+    t.integer "authorId"
+    t.integer "postId"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "posts", force: :cascade do |t|
+    t.integer "authorId"
+    t.string "title"
+    t.string "text"
+    t.integer "commentsCounter"
+    t.integer "likesCounter"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```

Comment:
- [ ] You are doing an excellent job, congrats on it! 🎉 However, it is a RoR convention to use only downcase and underscore for naming your tables and columns, for example instead of `postCounter` it would be better to use `post_counter`.

- [ ]  Posts, Comments and Likes tables have all the necessary columns but they are missing the foreign key, you can add a foreign key by using the command `rails generate migration AddUserRefToPosts user:references` ([source](https://edgeguides.rubyonrails.org/active_record_migrations.html)) or you can add it from the beginning at the model generation e.g: 
`rails generate model Post title:text user:references`
After doing this you should be able to see something like this in the schema file:
```
  create_table "posts", force: :cascade do |t|
    t.integer "authorId"
   ..
   t.references :author, foreign_key: { to_table: 'users' } 
  end
```